firefox-ios/Shared/Supporting Files/pt-PT.lproj/MainMenu.strings

```diff
@@ -11,7 +11,7 @@
 "MainMenu.Account.AccessibilityLabels.CloseButton.v132" = "Fechar";
 
 /* The accessibility label for the sign in button in the Main menu header view. */
-"MainMenu.Account.AccessibilityLabels.MainButton.v132" = "Iniciar sessão para sincronizar palavras-passe, separadores e mais";
+"MainMenu.Account.AccessibilityLabels.MainButton.v132" = "Iniciar sessão para sincronizar palavras-passe, separadores e muito mais";
 
```

Comment:
@Delphine Likewise some additional changed strings in this file...

Reply:
Also good! we can expect these kind of changes as we trust translators who have sufficient permissions to update translations.

Reply:
Perfect, thank you @Delphine! I'll keep this in mind in the future. 😄